src/config.cpp

```diff
@@ -629,16 +634,18 @@ void parseConditionsModifications(const nlohmann::json& it,
         case SimulationConfig::ModificationBase::ModificationType::TTX: {
             SimulationConfig::ModificationTTX result;
             result.type = type;
+            parseMandatory(valueIt, "name", debugStr, result.name);
             parseMandatory(valueIt, "node_set", debugStr, result.nodeSet);
-            buf[mIt.key()] = result;
+            buf.push_back(std::move(result));
```

Comment:
maybe use emplace_back instead?

Reply:
As we use `std::move` here, it avoids copy, right? Is `emplace_back` still better?

Reply:
you are right it is equivalent in this case...ignore please!